source/clear-linux/guides/tooling/autospec.rst

```diff
@@ -238,11 +238,11 @@ and dependencies in order for autospec to complete a successful build.
 
       cd packages/opae-sdk
 
-#. Add a license:
+#. Add a valid license identifier from the `SPDX License List <https://spdx.org/licenses/>`_:
 
    .. code-block:: bash
 
-      echo "Intel Corporation" > opae-sdk.license
+      echo "MIT" > opae-sdk.license
```

Comment:
Should be `BSD-3-Clause MIT`. See the [COPYING](https://github.com/OPAE/opae-sdk/blob/master/COPYING) file.

Reply:
I'm engaging legal to review this as well.

Reply:
Thanks @phmccarty for verifying it's `BSD-3-Clause MIT`.